market/auction.cpp

```diff
@@ -1598,8 +1598,8 @@ int auction::submit_nolock(char *from, double quantity, double real_price, KEY k
 				gl_name(OBJECTHDR(this),myname,sizeof(myname)),quantity<0?"ask":"offer",
 				from);
 		}
-		return 1;
 	}
+	return 1;
```

Comment:
I think this one should be return 0. It looks like a fall-through case.

Reply:
Changed in auction and auction_ccsi to return 0 in fall-through case. Thanks!